doc/tutorials/cpu_sharing.rst

```diff
@@ -126,29 +126,6 @@ and BVT (Borrowed Virtual Time) scheduler.
     timesharing.
 
 
-Scheduler configuration
```

Comment:
Need to add how to configure the scheduler instead of using Kconfig, something like:
```
The scheduler used at runtime is defined in the scenario XML file
via the :option:`hv.FEATURES.SCHEDULER` option.
The default scheduler is **SCHED_BVT**.
```

Reply:
@Kunhui-Li  Can you make this documentation change?

Reply:
Sorry for my later reply, updated it, thanks.